src/components/layout/Navigation.tsx

```diff
@@ -42,69 +36,29 @@ const Navigator = (): JSX.Element | null => {
     enabled: !!itemPath,
   });
 
-  const isParentOwned =
-    currentMember &&
-    (item?.creator?.id ?? parents?.[0]?.creator?.id) === currentMember?.id;
-
   if (isItemLoading || areParentsLoading) {
     return null;
   }
 
   const buildToItemPath = (id: string) => buildItemPath(id);
 
-  const menu = [
-    // todo: remove distinction -> not a good idea to show the whole root in arrow
-    {
-      name: translateBuilder(BUILDER.NAVIGATION_MY_ITEMS_TITLE),
-      id: 'home',
-      to: HOME_PATH,
-    },
-    {
-      name: translateBuilder(BUILDER.NAVIGATION_SHARED_ITEMS_TITLE),
-      id: 'shared',
-      to: SHARED_ITEMS_PATH,
-    },
-    {
-      name: translateBuilder(BUILDER.NAVIGATION_BOOKMARKED_ITEMS_TITLE),
-      id: 'bookmark',
-      to: BOOKMARKED_ITEMS_PATH,
-    },
-  ];
-
   const renderRoot = () => {
     // no access to root if signed out
     if (!currentMember) {
       return null;
     }
 
-    const selected =
-      isParentOwned || pathname === HOME_PATH ? menu[0] : menu[1];
-
     return (
       <>
-        <HomeMenu
-          selected={selected}
-          elements={menu}
-          buildMenuItemId={buildNavigationLink}
-        />
-        <ItemMenu
-          itemId="root"
-          useChildren={
-            isParentOwned || pathname === HOME_PATH
-              ? (useOwnItems as any)
-              : useSharedItems
-          }
-          buildToItemPath={buildToItemPath}
-        />
+        <IconButton id={NAVIGATION_HOME_ID} onClick={() => navigate(HOME_PATH)}>
```

Comment:
I like that the home is now clickable, but maybe we should use a react-router link passed in the component prop instead of a js navigate ? This would allow opening in a new tab etc.